mlrun/serving/remote.py

```diff
@@ -454,3 +459,67 @@ def _get_data(self, data, headers):
         ) and isinstance(data, (str, bytes)):
             data = json.loads(data)
         return data
+
+
+class MLRunAPIRemoteStep(RemoteStep):
+    def __init__(
+        self, method: str, path: str, fill_placeholders: Optional[bool] = None, **kwargs
+    ):
+        """
+        Graph step implementation for calling MLRun API endpoints
+
+        :param method:  HTTP method (GET, POST, ...) as default defines the method of the request, if empty string or
+                        None the method will be taken from the event (default to POST)
+        :param path:    API path (e.g. /api/projects)
+        :param fill_placeholders: if True, fill placeholders in the path using event fields
+        :param kwargs:  other arguments passed to RemoteStep
+        """
+        super().__init__(url="", method=method, **kwargs)
+        self.rundb = None
+        self.path = path
+        self.fill_placeholders = fill_placeholders or False
```

Comment:
The default actually shouldn't be set in the constructor, because it will cause `fill_placeholders` to be written as `False` when the class is serialized, rather than being omitted (when the user didn't specify it).